packages/@okta/vuepress-site/.vuepress/nav/guides.js

```diff
@@ -0,0 +1,44 @@
+// the navigation that is shown across the guides pages
+
+module.exports = [
+    { title: 'Add an external Identity Provider', guideName: 'add-an-external-idp' },
```

Comment:
Not sure how we'll want to implement it, but we'll want a `Guides Overview` link to https://developer.okta.com/docs/guides/ as the first item in the tree nav. (As we want `{section} Overview` links to the overview articles for each section.)

Reply:
Yeah I am not sure how to address this as clicking on parent expands parent item to show children. Do we want to have two actions(open the page and also expand the children) on single click? if we do that we will loose exploration of items without page refreshes.

Reply:
Hmm. I don't think we'd want two actions as a single click. Sub-sections (such as `Add an external Identity Provider`) should be purely expand/collapse groupings, not links that navigate the user.

The Reference section is probably a good example here. Some articles (say, `Error Codes`) are not in a sub-section, while some (say, `Manage Okta Resources > Apps`) are.

In Guides, `Guides Overview` should not be in a sub-section. We should support any hierarchy of articles – some not in a sub-section, some in a sub-section, some in nested sub-sections.

Reply:
I've added the Guides Overview link as the first item!